### ISSUE TITLE
Recipe for translated Enums

### ISSUE THREAD

**Issue by LourensVeen** [yatiml/yatiml#10]
The documentation covers Enum classes, and the Node class's documentation lets you get and set values on scalar nodes, but it would be nice to have a recipe for enums that use different strings in the YAML than for the value names on the Python side. It seems that PEP8 requires upper case Python names, while you may want to have lower case on the YAML side for instance.

**Comment by LourensVeen** [comment]
Or you may want to use the values, rather than the names of the attributes. The latter can be done like this:

```
class Genotype(enum.Enum):                                                      
    HOMO_NOSV = 'hmz'                                                          
    HOMO_SV = 'hmz-sv'                                                          
    HETERO_SV = 'htz-sv'                                                        
                                                                               
    @classmethod                                                                
    def yatiml_savorize(cls, node: yatiml.Node) -> None:                        
        yaml_to_python = {                                                      
                v._value_: v._name_ for v in cls.__members__.values()}          
                                                                               
        if node.is_scalar(str):                                                
            node.set_value(yaml_to_python.get(node.get_value()))                
                                                                               
    @classmethod                                                                
    def yatiml_sweeten(cls, node: yatiml.Node) -> None:                        
        node.set_value(cls.__members__[node.get_value()]._value_)      
```

This could be wrapped into some helper functions in yatiml.Node, so that these both become one-liners. And the same thing for uppercasing and lowercasing actually.

**Comment by LourensVeen** [comment]
Released in 0.6.0.
